### PR TITLE
LIVE-1960 Filecoin integration in LLD and LLM

### PR DESCRIPTION
**Pull request by haammar-ledger**
### ❓ Context

- **Impacted projects**: `@ledgerhq/live-common`, `live-desktop`, `live-mobile`
  
- **Linked resource(s)**: https://ledgerhq.atlassian.net/browse/LIVE-1960

### ✅ Checklist

- [x] **Test coverage**: _Did you write any tests to cover the changes introduced by this pull request?_
- [x] **Atomic delivery**: _Is this pull request standalone? In order words, does it depend on nothing else?_
- [x] **No breaking changes**: _Does this pull request contain breaking changes of any kind? If so, please explain why._

### CODE REVIEW

File: apps/ledger-live-desktop/src/renderer/drawers/OperationDetails/index.js [hunk 1]
```diff
@@ -641,16 +641,19 @@ type OperationDetailsExtraProps = {
...
-    <OpDetailsSection key={key}>
-      <OpDetailsTitle>
-        <Trans i18nKey={`operationDetails.extra.${key}`} defaults={key} />
-      </OpDetailsTitle>
-      <OpDetailsData>
-        <Ellipsis>{value}</Ellipsis>
-      </OpDetailsData>
-    </OpDetailsSection>
-  ));
+  return Object.entries(extra).map(([key, value]) => {
```
Comment (gre) [inline apps/ledger-live-desktop/src/renderer/drawers/OperationDetails/index.js:644]: generally, changes in the generic part must be shipped in a separate PR as a coin integration should only touch its "family". but to be check with others if it's acceptable for this one.

File: libs/ledger-live-common/src/bot/index.ts [hunk 2]
```diff
@@ -76,7 +76,7 @@ export async function bot({ currency, family, mutation }: Arg = {}) {
         if (mutation) {
           spec = {
             ...spec,
-            mutations: spec.mutation.filter((m) =>
+            mutations: spec.mutations.filter((m) =>
```
Comment (gre) [inline libs/ledger-live-common/src/bot/index.ts:79]: wow, is this a bug? separate PR maybe?

File: pnpm-lock.yaml [hunk 3]
```diff
@@ -29590,7 +29580,7 @@ packages:
     engines: {node: '>=8.0.0'}
     dependencies:
       eventemitter3: 4.0.7
-      follow-redirects: 1.15.0
+      follow-redirects: 1.15.0_debug@4.3.4
```
Comment (gre) [inline pnpm-lock.yaml:29583]: curious about the diff here.

File: libs/ledger-live-common/src/bot/index.ts (hunk 2 above)
Comment (haammar-ledger) [reply libs/ledger-live-common/src/bot/index.ts:79]: Yep, spotted and fixed by @emmanuelm41 :slightly_smiling_face: 
Here's a dedicated PR: https://github.com/LedgerHQ/ledger-live/pull/168
@gre no need to remove the fix from that PR right?

File: pnpm-lock.yaml (hunk 3 above)
Comment (haammar-ledger) [reply pnpm-lock.yaml]: (pnpm-lock.yaml updated)
